.changeset/young-items-drop.md

```diff
@@ -1,9 +1,10 @@
 ---
 "@medusajs/workflows": patch
 "@medusajs/product": patch
+"@medusajs/pricing": patch
```

Comment:
todo: There is two changeset, the first one seams targetting the right package, but not this one

Reply:
`pricing` was missing in the core integration changeset from the pr we merged earlier, just adding it here